doc/tutorials/tutorial_X8.rst

```diff
@@ -72,6 +74,14 @@ with the following restrictions:
 
 * Finally, the chip terminates with photon-number resolving measurements (:class:`~.MeasureFock`).
 
+.. figure:: /_static/X8.svg
```

Comment:
something seems wrong with the line about SU(4) at the bottom of the image

Reply:
The sentence "Programmable SU(4) transformation"? I can change that to "Arbitrary 4x4 unitary"(?)

This comes straight from the hardware documentation, I'm also worried the single mode squeezers followed by entanglers in the circuit diagram might be confusing; best to change this to two mode squeeze gates?

Reply:
I might merge this in now, and we can update the image in a new PR :)